target/linux/ipq40xx/image/generic.mk

```diff
@@ -1267,3 +1274,22 @@ define Device/zyxel_wre6606
 endef
 # Missing DSA Setup
 #TARGET_DEVICES += zyxel_wre6606
+
+define Device/zyxel_wsq50
+	DEVICE_VENDOR := ZyXEL
+	DEVICE_MODEL := WSQ50
+	DEVICE_ALT0_VENDOR := ZyXEL
+	DEVICE_ALT0_MODEL := WSQ60
+	SOC := qcom-ipq4019
+	KERNEL_SIZE := 4096k
+	BLOCKSIZE := 64k
```

Comment:
Are you sure?

That sounds really low for eMMC

Reply:
The BLOCKSIZE? Yes
The KERNEL_SIZE? Unsure, that was stolen from elsewhere

Reply:
Then you can drop `KERNEL_SIZE` as its unused

Reply:
This has been fixed - `KERNEL_SIZE` removed as robimarko was correct, it's superfluous